libnmstate/_error.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+
+class NmstateError(Exception):
+    BUG = -1
+    NOT_SUPPORT = 95
+
+    _LOCALS = locals()
```

Comment:
Can the error numbers maybe be collected in a sepreate enum class instead?

Reply:
Enum is new in python 3.4. Are we dropping the support of python2.7?

Reply:
Forgot that we need to support 2.7 here, too.